### DISCUSSION TITLE
Cannot add API rule to only view records made by authenticated user

### DISCUSSION THREAD

**Discussion by tphbrok** [pocketbase/pocketbase#496] [Q&A] [answered]
I assumed the following API rule would be valid:

`@request.user.id=user.id`

but I get an error stating that the rule is invalid.

The collection itself has two fields: a text field titled `name` and a relation field to `user`.

Am I conceptually misunderstanding the usage of an API rule or is this an actual bug?

![image](https://user-images.githubusercontent.com/11331876/190628943-e4607f99-0bcb-4dab-8309-f2de131c046d.png)

**Comment by ganigeorgiev** [comment, marked as answer]
The `user` type fields cannot be accessed in the same way as the `relation` type fields. In your example the correct filter should be:
```
@request.user.id = user
```

_In any case, please note that the `user` type field will be removed and we will have only the `relation` type field after #376._
